Form/Type/AceEditorType.php

```diff
@@ -9,30 +9,25 @@
  * file that was distributed with this source code.
  */
 
-namespace Norzechowicz\AceEditorBundle\Form\Extension\AceEditor\Type;
+namespace Norzechowicz\AceEditorBundle\Form\Type;
```

Comment:
You can have a look here https://insight.sensiolabs.com/projects/26aa2f34-a5bd-4c95-957d-558c5c28f0ac/analyses/1

Below your quote on the same page:
```
AcmeBundle/
    Form/
        ChoiceList/
            ColorChoiceList.php
        DataTransformer/
            RgbToHexTransformer.php
        Type/
            WidgetType.php
```

we see that the Type (and not the whole extension files) is in the `Type` directory